cmd/client-fs.go

```diff
@@ -942,8 +942,13 @@ func (f *fsClient) listRecursiveInRoutine(contentCh chan *clientContent) {
 }
 
 // MakeBucket - create a new bucket.
-func (f *fsClient) MakeBucket(region string) *probe.Error {
-	e := os.MkdirAll(f.PathURL.Path, 0777)
+func (f *fsClient) MakeBucket(region string, ignoreExisting bool) *probe.Error {
+	var e error
+	if ignoreExisting {
+		e = os.MkdirAll(f.PathURL.Path, 0777)
```

Comment:
This changes backward compatibility... Lets do 'MkdirAll` anyways until some release in future. Add a TODO to move it back to `Mkdir` in future.